src/rprblender/properties/render.py

```diff
@@ -444,7 +444,7 @@ def update_render_quality(self, context):
         name="Render Quality",
         description="RPR render quality",
         items=render_quality_items,
-        default='FULL',
+        default='FULL2' if pyrpr2.enabled else 'FULL',
```

Comment:
What do you think about removing pyrpr2.enabled and config.enable_rpr2 options at all in this PR?